### PR TITLE
Update black rev in order to include fix for https://github.com/psf/black/issues/1629

### PR DESCRIPTION
**Pull request by tafaRU**
I ran into this [error](https://github.com/tafaRU/l10n-italy/runs/4289069672?check_suite_focus=true#step:4:105) related to https://github.com/psf/black/issues/1629.
In order to fix it, black needs to be updated at least to https://pypi.org/project/black/21.4b0.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: 19.10b0
+    rev: 21.4b0
```
Comment (simahawk) [inline .pre-commit-config.yaml:3]: this is the repo's file. I guess you want to change this instead https://github.com/OCA/oca-addons-repo-template/blob/master/src/.pre-commit-config.yaml.jinja#L4 ?

Comment (tafaRU) [reply .pre-commit-config.yaml:3]: Ok, thanks for the advice @simahawk!

**Comment by sbidoul** [comment]
The problem with changing black on active branches is that it creates some churn with PRs becoming red due to minor reformating. That said, black is getting more and more stable over time.

Alternatives:
- using the `# fmt: on/off/skip` black directive to work around local issues
- making the black version a copier question (with same defaults as today), so we can let PSCs manage their black version autonomously across branches

**Comment by tafaRU** [comment]
Ok, thanks for the explanation.
In the meantime I found the lines could make black fail and I fix them. So I can proceed without this.

Personally I'd prefer the following:
> making the black version a copier question (with same defaults as today), so we can let PSCs manage their black version autonomously across branches

**Pull request closed without merging**